IDisposableAnalyzers/build/OwnershipAttributes.cs

```diff
@@ -0,0 +1,28 @@
+namespace IDisposableAnnotations
```

Comment:
What should the namespace be?

Reply:
Maybe `IDisposableAnalyzers` is clearest?

Reply:
Sounds good, updated.